contrib/platform/src/com/sun/jna/platform/win32/User32Util.java

```diff
@@ -155,7 +155,12 @@ public Object call() throws Exception {
         private volatile int nativeThreadId = 0;
         private volatile long javaThreadId = 0;
         private final List<FutureTask> workQueue = Collections.synchronizedList(new ArrayList<FutureTask>());
-        
+        private static long messageLoopId = 0;
```

Comment:
I don't see this as critical, but shouldn't this be an AtomicLong/AtomicInteger? The constructor could be invoked from different threads in parallel.